doc.html

```diff
@@ -0,0 +1,118 @@
+        h2 { margin: 10px 0 0 -22px; font-size:12px; padding: 0 !important }
+        table { font-size: 10px; margin-left: -22px; width: 480px; overflow-wrap: break-work; }
+        table td { overflow-wrap: break-work; }
+        div.header { padding: 0; margin: 0 0 0 -22px; white-space: pre-wrap; }
+    </style>
+</head>
+<body>
+
+<div class="header">
+ATLAS Service Template Specification
+Name                                                    feed_data
+Version                                                     0.0.3
+Author(s)             ? (MmmooOgle), J. V&ouml;lker (fodjan GmbH)
+</div>
+
+<h2>The "feed_data" ATLAS Service Template</h2>
+<div style="margin-left: -22px;">
+
+<h2>Abstract</h2>
+This document describes the specification for the feed_data service whose purpose is to provide an interface for data-exchange of planned and actual feeding.
+
+</div>
+<h2>1 Introduction</h2>
+Exchange planned rations and actual feedingreports is an important step to keep animals healthy, control the feedcosts and reduce environmental impact by increasing the precision of feeding.
+
+This service provide endpoints to read the planned rations out of the system and write back actual feeded amounts.
+
+<h2>2 Terminology</h2>
+
+The key words "MUST", "MUST NOT", "REQUIRED", "SHALL", "SHALL NOT", "SHOULD", "SHOULD NOT", "RECOMMENDED", "NOT RECOMMENDED", "MAY", and "OPTIONAL" in this document are to be interpreted as described in RFC 2119 and indicate requirement levels for compliant implementations.
+
+The notation "[xxx]" (xxx in square brackets) is equivalent to "array of xxx".
+
+When used alone, the term "feed_data" refers to "feed_data service template". Instead, "animal_data service" is equivalent to "a service implementing the animal_data service template".
+
+<h2>3 Pre-requisites</h2>
+
+The service has to manage rations and their usage dates.
+
+The value of the system increases with extended feed-models which can calculate figures like expected yield-output and methan-emissions based on the feedstuffs mixed in that ration.
+
+<h2>4 Use Scenarios</h2>
+
+There are two usage scenarios:
+- Send planned rations to service and over the service-owner to feed-wagons
+- Read actual feedingreports into the system
+
+<h2>5 Service Template API Overview</h2>
+
+2 Endpoints:
+
+- POST /ration: send a ration to the system
+- GET /feedingreport: return all feedingreports in the given time range
+
+<h2>6 Data Formats</h2>
+
+The following data model describes the 3 main entities and how they interact with the service animal_data.
+
+<img src="doc/atlas feedingmodel datamodel.png" style="width:450px" />
+
+<h3>Ration</h3>
+A ration is planned for a certain date and calculated for a single cow per day. This comes mostly out of ration calculation software and is used as starting point to programm a mixer wagon. There might be other entities like a pen included to calculate the real amount per group and per feeding.
```

Comment:
is the feeding_data service template specific to "corw" as suggested in this section?
If so, it should be renamed to cow_feeding_data. Otherwise rename "cow" to animal (here and wherever else cow may be mentioned)

Reply:
right, we should rename all occurences from cows to animal.